docs/migrations.rst

```diff
@@ -4,6 +4,55 @@ Migration Guide
 This page contains information about changes between major versions and
 how you can migrate from one version to another.
 
+0.13.x to 0.13.3
+----------------
+
+- ``rasa_nlu.server`` needs to be supplied with an ``yml`` file defining the
+	model endpoint to retrieve training data. The file location has to be passed
+	with the ``--endpoints`` argument, e.g.
+	``python rasa_nlu.server --path projects --endpoints endpoints.yml``
+	``endpoints.yml`` needs to contain the ``model`` key
+	with a ``url`` and an optional ``token``. Here's an example:
+
+	.. code-block:: yaml
+
+			model:
+				url: http://my_model_server.com/models/default/nlu/tags/latest
+				token: my_model_server_token
+
+- ``rasa_nlu.train`` also has to be run with the ``--endpoints`` argument
```

Comment:
I am not sure we should replace this syntax. I see the point about unifying the syntax, but then again the url one is simpler to share (e.g. in the platform)